she_codes_news/news/templates/news/index.html

```diff
@@ -1,21 +1,57 @@
-    <h2>Latest News</h2>
+    {% extends "base.html" %}
+    {% load static %}
 
-    <div id="latest-stories">
+    {% block stylesheets %}
+    <link rel="stylesheet" type="text/css" href="{% static 'news/news.css' %}">
+    {% endblock %}
+
+    {% block content %}
+    <div id="index-header">
+        <div id="index-header-2">
+            <div>
+                {% if request.user.is_authenticated %}
+                    <h2> Welcome {{ user.username }}!</h2>
+                    <a id="new-button" href="{% url 'news:newStory' %}">Write New Story</a>
+                {% endif %}
+            </div>
+            <div>
+                <h2 id="index-header-latest-news">The latest news from She Codes:</h2>
+            </div>
+        </div>
+
+        <div id="index-header-3">
+            <div>
+                <h4>Filter by category:</h4>
+                <div id="index-header-cats">
+                    {% for cat in categories %}
+                    <a class="category-button" href="{% url 'news:index' %}?cat={{cat.id}}">{{ cat.name }}{% endfor %}</a>
+                    <a class="category-button" href="{% url 'news:index' %}">All</a> 
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- <div id="latest-stories">
         {% if latest_stories %}
             {% for story in latest_stories %}
-                <p>{{ story.title }}</p>
+                {% include './storyCard.html' with story=story position=forloop.counter %}
             {% endfor %}
         {% else %}
             <p>No stories are available.</p>
         {% endif %}
-    </div>
+    </div> -->
```

Comment:
this html comment means all this code still gets generated on the backend, which can take up a fair bit of processing time, better to delete it, or use a django comment, rather than a html comment